Bottom: 668e4efd2c42c95930cf5b6b0a761c744d84751d
Top:    10289c38707fc85b4571c986efcb835301f37584
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2008-11-04 19:06:38 -0500

Refresh of jds-lirc-core

---

diff --git a/drivers/input/ir/ir-core.c b/drivers/input/ir/ir-core.c
index 1eff985..3a309fa 100644
--- a/drivers/input/ir/ir-core.c
+++ b/drivers/input/ir/ir-core.c
@@ -12,10 +12,13 @@
 
 void input_ir_translate(struct input_dev *dev, int protocol, int device, int command)
 {
-
+	/* generate the IR format event */
+	input_report_ir(dev, IR_PROTOCOL, protocol);
+	input_report_ir(dev, IR_DEVICE, device);
+	input_report_ir(dev, IR_COMMAND, command);
+	input_sync(dev);
 }
 
-
 static int encode_sony(struct ir_device *ir, struct ir_command *command)
 {
 	/* Sony SIRC IR code */
